### ISSUE TITLE
WoW Classic instance returning retail data

### ISSUE THREAD

**Issue by albri** [benweier/blizzard.js#123]
I'm trying to use this library specifically for WoW Classic as documented.

I am trying to create a WoW Classic instance like below

```
import { wow } from 'blizzard.js'

const wowClassicClient = await wow.classic.createInstance({
  ...
 })
```

But whenever I use wowClassicClient I only get retail specific data back.

For example ` await wowClassicClient.playableClass()` returns all 12 retail classes. Searching for items returns the retail versions.

Am I missing something somewhere?

Thanks

**Comment by benweier** [comment]
Hey @albri that's my bad I forgot to set the namespace of the wow classic methods. I'll get a fix going

**Comment by benweier** [comment]
@albri fixed, `4.1.0` is now published to npm

**Comment by albri** [comment]
Great, thanks a lot!
